site/template.html

```diff
@@ -138,6 +138,8 @@ <h5><a href=https://conda-static.anaconda.org/conda-forge/rss.xml>CDN cloning</a
         <h4 id="big_migrations"><a href="#big_migrations" style="color: inherit;">Long-Running Migrations</a></h4>
         <div id="migratorDivBig" style="text-align:center;"></div>
 
+        <h4><a href="#little_migrations" style="color: inherit;">More Migrations</a></h4>
```

Comment:
We should have the text be "Other Migrations". Also I think this a tag is a duplicate now.

Reply:
There are also "Recently Closed Migrations" in that section. Though no strong feelings

Reply:
Oh sorry. I was saying the page should look like it did before but when you click the links, instead of opening up the section, it would scroll down to the new place.

Reply:
```suggestion
        <h4><a href="#little_migrations" style="color: inherit;">Other Migrations</a></h4>
```